includes/admin/views/html-admin-page-status.php

```diff
@@ -35,7 +35,11 @@
 				WC_Admin_Status::status_logs();
 			break;
 			default :
-				WC_Admin_Status::status_report();
+				if ( array_key_exists( $current_tab, $tabs ) && has_action( 'woocommerce_admin_status_' . $current_tab ) ) {
```

Comment:
Technically you couldn't have a tab named `tabs` because the hook would conflict with the one above. Minor and edge-case I know, but should probably avoid a hook conflict like that.